.github/workflows/lint.yml

```diff
@@ -38,9 +38,9 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
```

Comment:
```suggestion
          node-version: 18.x
```

Reply:
fixed in the next PR 🐸👍

Reply:
fixed in https://github.com/defi-wonderland/solidity-foundry-boilerplate/pull/33/